Firestore/Example/Firestore.xcodeproj/xcshareddata/xcschemes/Firestore_Tests_iOS.xcscheme

```diff
@@ -20,20 +20,6 @@
                ReferencedContainer = "container:Firestore.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
```

Comment:
There is something I don't understand about this change and the Firestore schemes.  SwiftBuildTest is both in its own scheme and in the Firestore_Tests_iOS scheme. build.sh builds them both - https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh#L218. Therefore I would think the SwiftBuild test would still be failing - but Travis succeeds, so what am I missing?

Reply:
We have two Swift targets:

  * Firestore_SwiftTests_iOS, which requires Swift 4, because it's intended to exercise Codable stuff; and
  * SwiftBuildTest, which only requires Swift 3, because it tests the Firestore API and AFAIK we still support Swift 3 for that.

I removed the Firestore_SwiftTests_iOS target from Firestore_Tests_iOS scheme, making that scheme no longer require Swift 4. SwiftBuildTest is unaffected but still works because it only needs Swift 3.

Reply:
Thanks. LGTM 👍